### PR TITLE
Add other prometheus monitoring components

### PR DESCRIPTION
**Pull request by serathius**
This PR adds core prometheus monitoring components like alertmanager, kube-state-metrics, node exporter.
Configuration for manifests is extracted from Helm manifests.
```release-note
NONE
```
/cc @kawych @brancz

### CODE REVIEW

File: cluster/addons/prometheus/alertmanager-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: alertmanager
```
Comment (brancz) [inline cluster/addons/prometheus/alertmanager-deployment.yaml]: This is another reason why the Prometheus Operator is a good idea here. Running the alertmanager as a deployment is a bad idea, it delivers notifications about alerts and therefore needs to be the most reliable thing in your cluster. With a simple `Deployment` you're not getting any of the high availability features that you want, such as persistent storage and controlled rolling deployments.

File: cluster/addons/prometheus/alertmanager-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    k8s-app: alertmanager
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+    version: v0.13.0
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: alertmanager
+      version: v0.13.0
```
Comment (brancz) [inline cluster/addons/prometheus/alertmanager-deployment.yaml]: Latest stable release is v0.14.0

File: cluster/addons/prometheus/kube-state-metrics-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
+apiVersion: apps/v1beta2
```
Comment (brancz) [inline cluster/addons/prometheus/kube-state-metrics-deployment.yaml]: we're mixing the `apiVersion`s here, we should be consistent

**Review by brancz — CHANGES_REQUESTED** [review]
Regarding the Alertmanager, I would like us to seriously consider using the Prometheus Operator in this setup, there are a lot of factors like the ones described in my comment. Those are the kind things that we have been doing for the past two years, and it feel we should seriously consider using it.

**Comment by serathius** [comment]
As currently prometheus addon is in experimental state, we don't want/need to introduce all prometheus operator features at once.
@brancz do you have any restrictions about starting with simpler helm setup, without going full HA for AlertManager?

**Comment by brancz** [comment]
With it only being for testing purposes now I’m ok with this (although a short discussed even this should eventually move out of the Kubernetes repo)

/lgtm

**Comment by serathius** [comment]
I rebased it to master

**Pull request merged**